src/modules/drawing/components/ExtendLineInteraction.vue

```diff
@@ -0,0 +1,25 @@
+<script setup>
+import Feature from 'ol/Feature'
+
+import useExtendLineInteraction from '@/modules/drawing/components/useExtendLineInteraction.composable'
+const props = defineProps({
+    startingFeature: Feature,
+})
+const emits = defineEmits({
+    drawEnd(payload) {
+        return payload instanceof Feature
+    },
+})
+const { removeLastPoint } = useExtendLineInteraction({
+    drawEndCallback: (feature) => {
+        emits('drawEnd', feature)
+    },
+    startingFeature: props.startingFeature,
```

Comment:
I'm not sure it's a good idea to give that without wrapping it in a `ref`. The composable down the line isn't called once more when this value changes (if you select another line of the same drawing), so reactivity might be lost here.
But after testing, it seems to work fine...

Reply:
I handled the selection in `DrawingInteractions.vue`, see [this](https://github.com/geoadmin/web-mapviewer/pull/1177/commits/7d885b6874f700975b888d9412697e03d9ade4a6#diff-6cb2ff831d9178e2c52712eb1b44e2437ba8c6b600351228eaed444ad8a13d3aR79-R91). Basically, if the feature selected is changed (from the select interaction) it will update the selectedLineFeature.

Or am I wrong?